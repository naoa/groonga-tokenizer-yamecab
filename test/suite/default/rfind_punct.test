register tokenizers/yamecab

config_set tokenizer-yamecab.parse_limit 19

tokenize TokenYaMecab "今日は、雨です"
