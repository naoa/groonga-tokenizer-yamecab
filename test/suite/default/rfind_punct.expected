register tokenizers/yamecab
[[0,0.0,0.0],true]
load --table @yamecab
[
{"_key": "TokenYaMecab", "parse_limit":19, "@updates":true}
]
[[0,0.0,0.0],1]
tokenize TokenYaMecab "今日は、雨です"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "今日",
      "position": 0
    },
    {
      "value": "は",
      "position": 1
    },
    {
      "value": "、",
      "position": 2
    },
    {
      "value": "雨",
      "position": 3
    },
    {
      "value": "です",
      "position": 4
    }
  ]
]
